examples/api-call/manifest.json

```diff
@@ -0,0 +1,47 @@
+{
+  "version": "0.1.0",
+  "createdAt": "2022-07-26T12:51:00.000000Z",
+  "expiresAt": "2100-01-01T00:01:00.000000Z",
+  "metadata": {
+    "name": "External API call example",
+    "description": "Example manifest of a service making an outbound call to the external API",
+    "version": "0.1.0"
+  },
+  "payload": [
+    {
+      "platform": {
+        "arch": "x86_64",
+        "os": "linux"
+      },
+      "urls": [
+        "http://yacn2.dev.golem.network:8000/docker-golem-curl-jq-latest-9a163ce9d3.gvmi"
+      ],
+      "hash": "sha3:5f315c3756ec14a9dd4ed1a230ab4bbaabcc813d979e3f11cefd5458"
+    }
+  ],
+  "compManifest": {
+    "version": "0.1.0",
+    "script": {
+      "commands": [
+        "run .*",
+        "transfer .*"
```

Comment:
We don't use transfer, so we could remove this line

Reply:
I had no idea what it does, and forgot to change it. Transfer got removed.